### PR TITLE
Doc updates for the Sandboxed Extensions Dev environment

### PR DESCRIPTION
**Pull request by d45**
@brtal @seanmakesgames 
This PR includes updates to the Readme.md file and new files for the Sandboxed Extensions dev environment.

### CODE REVIEW

**Comment by d45** [comment]
@seanmakesgames  I'm not sure why this PR includes the commits for the sample .css and .js files, and the package.json.  I'll defer to you for those files.  Thanks.

File: Samples/UINamespace-sandboxed/uiNamespace.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```
Comment (seanmakesgames) [inline Samples/UINamespace-sandboxed/uiNamespace.html]: Something is funky. This should not be showing up in the diff.

Comment (seanmakesgames) [reply Samples/UINamespace-sandboxed/uiNamespace.html]: Ok fixed. :|

File: docs/trex_sandbox_test.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+To port an existing extension, make sure all resources and libraries are local to your extension source code. No network access outside the sandbox container is allowed. And you need to modify the path to your extension in the `.trex` file to have it load in the sandbox environment. Information about changing your `.trex` file is described in [Test your own Sandboxed Extensions in the local sandbox](#test-your-own-sandboxed-extensions-in-the-local-sandbox).
+
+
+### Requirements for Sandboxed Extensions
+
+* Sandboxed Extensions cannot make calls to remote or external external resources or libraries. All content must be local to the extension. For example, if your extension uses the JQuery library, the library can't be accessed via HTTPS (`src="https://code.jquery.com/jquery-3.2.1.min.js"`). The library must be downloaded and on the same computer as the extension. All links must be relative (`src="./jquery-3.2.1.min.js`).
```
Comment (brtal) [inline docs/trex_sandbox_test.md]: external is repeated in the first sentence

Comment (seanmakesgames) [reply docs/trex_sandbox_test.md]: they are VERY external resources! ;)

Comment (d45) [reply docs/trex_sandbox_test.md]: Oh s* Whoops! Must have been an external error.  Thanks for catching that. 
It's fixed now.

**Pull request merged**